packages/telerivet/src/Adaptor.js

```diff
@@ -29,10 +24,12 @@ export function execute(...operations) {
 
 /**
  * Send a message
- * @example
- * execute(
- *   send(data)
- * )(state)
+ * @example <caption>Send a message</caption>
+ * send({
```

Comment:
Just for future reference, if we're not sure about what sort of data to put in an example, I think `send($.data)` is absolutely fine. Or even like:
```
send({
   to_number: $.targetTelNumber,
   message: $.message,
   route_id: $.route
})
```

We know that most users will be passing (or transforming) data from state into operations. It's probably quite rare to actually hard-code strings like this. So having an example which uses state is just as likely to be useful, and saves us making up plausible data.

No change needed here - just something to bear in mind for future docs.

Reply:
Noted 👍🏽